var/spack/repos/builtin/packages/superlu-dist/package.py

```diff
@@ -32,6 +32,7 @@ class SuperluDist(Package):
     homepage = "http://crd-legacy.lbl.gov/~xiaoye/SuperLU/"
     url = "http://crd-legacy.lbl.gov/~xiaoye/SuperLU/superlu_dist_4.1.tar.gz"
 
+    version('xsdk-0.1.0', git='https://github.com/xiaoyeli/superlu_dist', commit='0b5369f')
```

Comment:
@tgamblin :  How about I strip multiple infinity-like versions from https://github.com/LLNL/spack/pull/1983 but keep the rest, it would allow you write here `develop.xdsk.0.1.0` and it will behave correctly w.r.t comparision with `5.0.0`?